The .man files are never installed so there's no point in 
generating them, especially since the source manpages are 
installed.

--- mdadm-1.9.0/Makefile
+++ mdadm-1.9.0/Makefile
@@ -69,3 +69,3 @@
 
-all : mdadm mdadm.man md.man mdadm.conf.man
+all : mdadm
 
